plugins/riemann-controller/riemann_controller/resources/manager.config

```diff
@@ -77,7 +77,9 @@
 
 (defn- amqp-connection-conf
   []
-  {:host (or (System/getenv "RABBITMQ_HOST") "localhost")})
+  {:host (or (System/getenv "RABBITMQ_HOST") "localhost")
+   :username (or (System/getenv "RABBITMQ_USER") "testuser")
```

Comment:
Created for backward compatibility again (and again broken by changing it from guest/guest... very smart of me).